### PR TITLE
feat(tutorial): complete step 2

### PR DESCRIPTION
**Pull request by wata-nbkn**
Closes #

{{short description}}

#### Changelog

**New**

- {{new thing}}

**Changed**

- {{change thing}}

**Removed**

- {{removed thing}}

### CODE REVIEW

File: src/content/LandingPage/_landing-page.scss [hunk 1]
```diff
@@ -0,0 +1,54 @@
+@import 'carbon-components/scss/globals/scss/vendor/@carbon/type/scss/font-family.scss';
+@import 'carbon-components/scss/globals/scss/vendor/@carbon/layout/scss/breakpoint.scss';
+@import 'carbon-components/scss/globals/scss/typography.scss';
+@import 'carbon-components/scss/globals/scss/vars.scss';
```
Comment (kellychurchill) [inline src/content/LandingPage/_landing-page.scss]: These should have been added to `src/app.scss`

Comment (wata-nbkn) [reply src/content/LandingPage/_landing-page.scss]: @kellychurchill Fixed.

File: src/index.scss [hunk 2]
```diff
@@ -26,3 +26,8 @@ main {
 .bx--content {
   padding: 0;
 }
+
+$feature-flags: (
+  ui-shell: true,
+  grid-columns-16: true,
+);
```
Comment (kellychurchill) [inline src/index.scss]: The $feature-flags mapping should come before the styles import. See https://www.carbondesignsystem.com/tutorial/react-step-2/#install-grid

Comment (kellychurchill) [reply src/index.scss]: Your Repos table width is not full width and your illustration on the landing page is not formatted properly. 
Please be sure to review your changes locally before pushing your updates.

Comment (wata-nbkn) [reply src/index.scss]: I'm sorry for the inconvenience. I fixed the style and verified the UI is formatted properly.

**Pull request closed without merging**